hbase-client/src/main/java/org/apache/hadoop/hbase/client/RawAsyncTableImpl.java

```diff
@@ -358,7 +358,7 @@ public CompletableFuture<Boolean> thenPut(Put put) {
         () -> RawAsyncTableImpl.this.<Boolean> newCaller(row, put.getPriority(), rpcTimeoutNs)
           .action((controller, loc, stub) -> RawAsyncTableImpl.mutate(controller, loc, stub, put,
             (rn, p) -> RequestConverter.buildMutateRequest(rn, row, family, qualifier, op, value,
-              null, timeRange, p, HConstants.NO_NONCE, HConstants.NO_NONCE),
+              null, timeRange, p, HConstants.NO_NONCE, HConstants.NO_NONCE, false),
```

Comment:
These methods correspond with the deprecated `CheckAndMutateBuilder`, so I didn't think it was worth making any client-side modifications to get this feature working with the deprecated builder.

Reply:
A carrot to encourage folks to move on :)